crates/orchestrator/Cargo.toml

```diff
@@ -6,3 +6,19 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+configuration = { path = "../configuration" }
+support = { path = "../support" }
+provider = { path = "../provider" }
+tokio = { workspace = true, features = ["time"] }
+thiserror = { workspace = true }
+# TODO: add logger in a new pr.
+#log = { workspace = true }
+multiaddr = { workspace = true }
+rand = "0.8"
+sha2 = { version = "0.10.2", default-features = false }
+hex = "0.4"
+sp-core = "22.0.0"
+serde_json = { workspace = true }
+futures = { workspace = true }
+libp2p = { version = "0.52" }
```

Comment:
We should probably hoist all dependencies to workspace Cargo.toml to enable other crates to have synchronized versions, it doesn't cost anything and make things cleaner imho, wdyt ?

Reply:
Yes, we should move it to the main Cargo.toml and use only the needed here. I will change it today.